buildroot-external/board/hardkernel/odroid-m1/hassos-hook.sh

```diff
@@ -3,18 +3,12 @@
 
 function hassos_pre_image() {
     local BOOT_DATA="$(path_boot_dir)"
-    local SPL_IMG="$(path_spl_img)"
 
     cp "${BINARIES_DIR}/boot.scr" "${BOOT_DATA}/boot.scr"
     cp "${BINARIES_DIR}"/*.dtb "${BOOT_DATA}/"
 
     cp "${BOARD_DIR}/boot-env.txt" "${BOOT_DATA}/haos-config.txt"
     cp "${BOARD_DIR}/cmdline.txt" "${BOOT_DATA}/cmdline.txt"
-
-    # SPL
-    create_spl_image
-
-    dd if="${BINARIES_DIR}/u-boot-rockchip.bin" of="${SPL_IMG}" conv=notrunc bs=512 seek=64
```

Comment:
It seems `image-spl-spl.cfg` and `partition-spl-spl.cfg` are missing for m1/m1s?

Reply:
Yep, noticed it too shortly before you: https://github.com/home-assistant/operating-system/pull/3388#discussion_r1611245763